src/components/projects/Projects.jsx

```diff
@@ -30,58 +30,57 @@ function ProjectList() {
   function handleEntryClick() {
     history.push("/projects/entry");
   }
-  function handleViewClick(id) {
-    history.push(`/projects/${id}`);
-  }
-  return (
-    <div className="Projects-Container">
-      <Button color="primary" onClick={handleEntryClick} variant="contained">
-        New Project
-      </Button>
 
-      <div className="Projects-List">
-        <TableContainer
-          component={Paper}
-          style={{ maxWidth: "1000px", minWidth: "500px" }}
-        >
-          {projects.length === 0 ? (
-            <div className="empty-file">
-              <DescriptionRoundedIcon
-                style={{ color: "#e2e2e2", fontSize: "10rem" }}
-              />
-              <Typography
-                style={{
-                  color: "#d2d2d2",
-                  fontSize: "1.2rem",
-                  fontWeight: "bold",
-                }}
-              >
-                Looks like, there are no projects
-              </Typography>
+  return (
+    <div className="projects-container">
+      <div className="top-content">
+        <Button color="primary" onClick={handleEntryClick} variant="contained">
+          New Project
+        </Button>
+        <div className="search-box">
+          <div className="search-box-input">
+            <InputBase placeholder="Search project…" />
+          </div>
+          <div className="search-box-icon">
+            <SearchIcon style={{ color: "#ffffff" }} />
+          </div>
+        </div>
+      </div>
+      <div className="projects-list">
+        {projects.length === 0 ? (
+          <div className="empty-file">
+            <DescriptionRoundedIcon
+              style={{ color: "#e2e2e2", fontSize: "10rem", margin: "30px" }}
```

Comment:
can we change this for classes? check this
{ makeStyles } from '@material-ui/core/styles';
*Note:  for all those components on the updated files